report/renderer/util.js

```diff
@@ -108,6 +108,22 @@ class Util {
     /** @type {Map<string, Array<LH.ReportResult.AuditRef>>} */
     const relevantAuditToMetricsMap = new Map();
 
+    // Old LHR versions (<=8.6.0) do not have the "hidden" group.
+    // This backcompat converts the LHR to use the new "hidden" group.
+    const [major, minor] = clone.lighthouseVersion.split('.').map(Number);
```

Comment:
let's describe what "no group" meant before vs what it means now in a comment here